src/components/elements/favorites-list.tsx

```diff
@@ -0,0 +1,103 @@
+"use client";
+
+import useFavorites from "@lib/hooks/useFavorites";
+import { ChatBubbleLeftEllipsisIcon, ClipboardDocumentIcon, EnvelopeIcon, HeartIcon} from "@heroicons/react/24/outline";
+import {useIsClient} from "usehooks-ts";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+
+
+const ShareButtons = () => {
+  const baseUrl = window.location.origin;
+  const { favs } = useFavorites();
+  const urlParams = `/favorites?fav=${favs.map(fav => `${fav.uuid}:${fav.title}:${fav.units}`).join(",")}`;
+
+  const handleCopy = async () => {
+    try {
+      await navigator.clipboard.writeText(baseUrl + urlParams);
```

Comment:
Kat mentioned wanting to include a popup that informs the user that the link has been copied to their clipboard but she is unsure about the accessibility requirements around that popup message 🤔

Reply:
Let's use `useCopyToClipboard()`
https://usehooks-ts.com/react-hook/use-copy-to-clipboard